hera_cal/abscal.py

```diff
@@ -2020,8 +2020,9 @@ def match_times(datafile, modelfiles, filetype='uvh5', atol=1e-5):
 
     # shift model files relative to first file & first index if needed
     for ml in model_lsts:
-        if ml[0] < model_lsts[0][0]:
-            ml += 2 * np.pi
+        for i in range(len(ml)):
```

Comment:
i'm generally wary of changing functions in other parts of the codebase not directly related to the PR, as they can have unintended, reverberating impacts--can you explain the motivation here? if the issue is a LST wrap then the easier thing to do is to just ensure each ml is unwrapped, I think

Reply:
Sorry about this, this was actually part of #757 but I wanted to include it here since I'm currently running the pipeline with this branch and need the bug fixed here too. #757 will be merged before this PR. I've added a dependency note to the PR description.

Reply:
This is no longer a new addition with the merger of #757